ocs_ci/deployment/cnv.py

```diff
@@ -792,3 +797,92 @@ def uninstall_cnv(self, check_cnv_installed=True):
 
         logger.info("Removing the openshift virtualization CRDs")
         self.remove_crds()
+
+    def cnv_patch_subscription(self, patch):
+        patch_cmd = (
+            f"oc -n {self.namespace} patch sub {constants.KUBEVIRT_HYPERCONVERGED} "
+            f"-p {patch} --type merge"
+        )
+        run_cmd(patch_cmd)
+
+    def get_running_cnv_version(self):
+        """
+        Get the currently deployed cnv version
+
+        Returns:
+            string: cnv version
+
+        """
+        occmd = (
+            f"oc get sub {constants.KUBEVIRT_HYPERCONVERGED} -n openshift-cnv -o json"
+        )
+        jq_cmd = "jq -r .status.currentCSV"
+        json_out = subprocess.Popen(shlex.split(occmd), stdout=subprocess.PIPE)
+        cnv_version = subprocess.Popen(
+            shlex.split(jq_cmd), stdin=json_out.stdout, stdout=subprocess.PIPE
+        )
```

Comment:
not critical but you can use `exec_cmd(cmd, shell=True) `it will save 5 lines of code and reduce complexity

Reply:
One more thing, as we are now not exporting the KUBECOFNIG to env variables on framework level, and it's handled in commands like exec_cmd or run_cmd where we automatically expose this to command and env variable on subprocess level. When you directly use subprocess function like Popen , you will have to handle yourself to propatate kubeconfig to the command itself. You can get the kubeconfig path from : `config.RUN["kubeconfig"]`

Reply:
will address this later.